perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -118,7 +113,13 @@
 
 
 FLAGS_IGNORED_FOR_CUSTOM_JOBFILE = {
-    'generate_scenarios', 'io_depths', 'run_for_minutes'}
+    'fio_generate_scenarios', 'fio_io_depths', 'fio_run_for_minutes'}
+
+
+# Modes from --fio_target_mode
+AGAINST_DEVICE_MODES = {'against_device_with_fill',
+                        'against_device_without_fill'}
+FILL_TARGET_MODES = {'against_device_with_fill', 'against_target_with_fill'}
```

Comment:
I think you meant "against_file_with_fill".

Please make sure you have COMPLETE coverage of this change which is relatively simple but more risky. (Things like this should have been caught by your testing effort)